x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -53,22 +54,18 @@ const msTimeUnitLabel = i18n.translate(
 );
 
 export class TransactionCharts extends Component<TransactionChartProps> {
-  public getResponseTimeTickFormatter = (t: number | null) => {
-    return this.props.charts.noHits ? `- ${msTimeUnitLabel}` : asMillis(t);
+  public getResponseTimeTickFormatter = (t: number | null | undefined) => {
+    return isValidCoordinateValue(t) ? asMillis(t) : `- ${msTimeUnitLabel}`;
```

Comment:
```suggestion
    return asMillis(t);
```

I don't see how react-vis will return an invalid coordinate (that would fail earlier afaict)